frontend/src/metabase/query_builder/components/filters/pickers/SelectPicker.jsx

```diff
@@ -130,6 +130,7 @@ export default class SelectPicker extends Component {
                                    <label className="flex align-center cursor-pointer p1" onClick={() => this.selectValue(option.key, !checked.has(option.key))}>
                                        <CheckBox
                                            checked={checked.has(option.key)}
+                                           onChange={() => {}}
```

Comment:
probably an accidental change?

Reply:
Sadly no, `CheckBox` requires a `onChange` handler to be passed. Since we're using the surrounding label as our toggle handler we don't require a `onChange` to be defined on the `CheckBox` but the `CheckBox` component does require it. I'm happy to look into removing the `onChange` handler requirement from the `CheckBox` component!

Reply:
Looks like we actually check for `props.onClick` before calling it, so all you should need to do is remove `isRequired` from the propType.